makefiles/tools/openocd.inc.mk

```diff
@@ -8,6 +8,9 @@ export DEBUGGER_FLAGS ?= debug
 export DEBUGSERVER_FLAGS ?= debug-server
 export RESET_FLAGS ?= reset
 
+# Include SERIAL variable compatibility code here
+include $(RIOTMAKE)/tools/serial_compat.inc.mk
```

Comment:
Same here, it is always included after `serial.inc.mk`.

If the compatibility should be kept it would make more sense to put it before including `$(BOARD)/Makefile.include`.